torch/jit/__init__.py

```diff
@@ -571,6 +584,8 @@ def trace(func, example_inputs, optimize=True, check_trace=True, check_inputs=No
         check_tolerance (float, optional): Floating-point comparison tolerance to use in the checker procedure.
                                            This can be used to relax the checker strictness in the event that
                                            results diverge numerically for a known reason, such as operator fusion.
+        try_outplace (bool, optional): If set, then the tracer will attempt to convert in-place ops to their
```

Comment:
This should be an internal argument

Reply:
What do you mean by "internal"?

Reply:
Shouldn’t be mentioned in the docstring, preferably prefixed with an underscore.